src/components/starRating/starRating.js

```diff
@@ -0,0 +1,18 @@
+import React,{Component} from 'react';
+import './starRating.scss'
+class StarRating extends Component {
```

Comment:
How will the rating be made dynamic?? 
Cause you'll be getting values like 3.5, 4.5 for rating from the API